api/src/test/java/org/openmrs/api/ObsServiceTest.java

```diff
@@ -1554,8 +1554,8 @@ public void saveObs_shouldCascadeUpdateToNewChildObsGroups() {
 		Obs origParentObs = obsService.getObs(2);
 		Set<Obs> originalMembers = new HashSet<>(origParentObs.getGroupMembers(true));
 		assertEquals(3, originalMembers.size());
-		assertTrue(originalMembers.contains(obsService.getObs(9)));
```

Comment:
Why do we change the order of these lines?

Reply:
I am really sorry for this. I did this by mistake. Forgot to change it back.